### ISSUE TITLE
`--ifdef no-indent` rule adds extra indent.

### ISSUE THREAD

**Issue by dollar2048** [nicklockwood/SwiftFormat#1448]
I have this piece of Swift code:

```swift
struct FontKit {
    var style: UIFont.TextStyle
    var size: CGFloat?
}

var title: FontKit {
    #if os(iOS)
    .init(style: .title2)
    #else
    .init(style: .title2, size: 40)
    #endif
}
```
When running SwiftFormat 0.51.9 with `--ifdef no-indent` I get:

```swift
struct FontKit {
    var style: UIFont.TextStyle
    var size: CGFloat?
}

var title: FontKit {
    #if os(iOS)
    .init(style: .title2)
    #else
        .init(style: .title2, size: 40)
    #endif
}
```
But I would expect no changes to be made.

Some interesting info: 
The following UnitTests (SwiftUI) are working fine:
https://github.com/nicklockwood/SwiftFormat/blob/master/Tests/RulesTests%2BIndentation.swift#L2135

So in case of some code presence before `#if` it works as expected.
Otherwise, not.

**Comment by nicklockwood** [comment]
@dollar2048 fixed in 0.51.10

**Comment by dollar2048** [comment]
Thank you @nicklockwood. it works now perfectly!
